database/migrations/2019_10_23_142210_create_faults_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFaultsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('faults', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('description');
+            $table->set('status', ['unseen', 'seen', 'done', 'wont_fix']);
```

Comment:
These statuses could be added as constants in the model (like in [PrintJob.php](https://github.com/luksan47/mars/blob/master/app/PrintJob.php))

Reply:
Are you sure, this should be done?
Constants aren't used when creating the print jobs table ([see here](https://github.com/luksan47/mars/blob/master/database/migrations/2019_10_06_224327_create_print_jobs_table.php)).

Reply:
Good catch! I think still that this should be fixed. Could you open an issue?
A better example is mac addresses: https://github.com/luksan47/mars/blob/master/database/migrations/2019_10_06_212436_create_mac_addresses_table.php

The reason behind using constants is that it eliminates the possible bugs from typos.

Reply:
OK, made the requested changes and opened issue for print jobs.
(Although mac addresses aren't perfect either, eg. constants aren't used [here](https://github.com/luksan47/mars/blob/master/app/Http/Controllers/InternetController.php#L165).)